nilearn/plotting/matrix/_utils.py

```diff
@@ -15,7 +23,95 @@ def _mask_matrix(mat, tri):
     return np.ma.masked_array(mat, mask)
 
 
-def _reorder_matrix(mat, labels, reorder):
+def pad_contrast_matrix(contrast_def, design_matrix):
```

Comment:
The test fail because I moved `pad_contrast_matrix` from `matrix_plotting` to `_utils`. So it is no more public. (I tried to move all functions that do not require `matplotlib` to _utils.)

Is there an inconvenience about it? It is not used elsewhere. If that is ok with you, I'll update the number of public functions.